Makefile

```diff
@@ -370,6 +370,12 @@ contracts/test/prover/proofs/%.json: $(arbitrator_cases)/%.wasm $(arbitrator_pro
 .make:
 	mkdir .make
 
+generate-hotshot-binding:
```

Comment:
Already exists in this file 335L.

Reply:
Gotcha - looks like the paths are different (HotShot is now stored in arbos/espresso/HotShot) looks like we don't use espressogen anymore so I think I can remove the old one